demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -534,6 +511,32 @@ def has_no_duplicate_args(self):
 
         return does_not_have_duplicate_args
 
+    def no_incident_in_core_packs(self):
+        """check if commands' name or argument contains the word incident"""
+
+        commands = self.current_file.get('script', {}).get('commands', [])
+        commands_with_incident = []
+        args_with_incident = {}
+        no_incidents = True
+        for command in commands:
+            command_name = command.get('name', '')
+            if 'incident' in command_name:
+                commands_with_incident.append(command_name)
+            args = command.get('arguments', [])
+            for arg in args:
+                arg_name = arg.get("name")
+                if 'incident' in arg_name:
+                    args_with_incident[command_name] = arg_name
```

Comment:
in case there are several arguments containing the word `incident`
```suggestion
                    args_with_incident.setdefault(command_name, []).append(arg_name)
```